<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.telkomsel.android.vmp.test.Group1_Regression_Post_NewTestCases_AdditionalTC" tests="1" time="188.730" ignored="0" failures="0" timestamp="07 Oct 2019 11:06:32 GMT" skipped="0" hostname="L-156182171" errors="1">
  <testcase name="Group1_Regression_NewTest_Cases" time="188.730" classname="com.telkomsel.android.vmp.test.Group1_Regression_Post_NewTestCases_AdditionalTC">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.telkomsel.android.vmp.pageobjects.SideBarMenuPage.verifylogout(SideBarMenuPage.java:296)
	at com.telkomsel.android.vmp.test.Group1_Regression_Post_NewTestCases_AdditionalTC.Group1_Regression_NewTest_Cases(Group1_Regression_Post_NewTestCases_AdditionalTC.java:84)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Group1_Regression_NewTest_Cases -->
</testsuite> <!-- com.telkomsel.android.vmp.test.Group1_Regression_Post_NewTestCases_AdditionalTC -->
